lib/machete/matchers/unix_time_matcher.ex

```diff
@@ -0,0 +1,119 @@
+defmodule Machete.UnixTimeMatcher do
+  @moduledoc """
+  Defines a matcher that matches integers that represent Unix time in milliseconds
+  """
+
+  import Machete.Mismatch
+
+  defstruct exactly: nil, roughly: nil, before: nil, after: nil
+
+  @typedoc """
+  Describes an instance of this matcher
+  """
+  @opaque t :: %__MODULE__{}
+
+  @typedoc """
+  Describes the arguments that can be passed to this matcher
+  """
+  @type opts :: [
+          {:exactly, Time.t()},
+          {:roughly, Time.t() | :now},
+          {:before, Time.t() | :now},
+          {:after, Time.t() | :now}
+        ]
+
+  @doc """
+  Matches against integers that represent Unix time values
```

Comment:
```suggestion
  Matches against integers that represent Unix time values in milliseconds
```

Reply:
(These docs are what users will be shown by LSP, so it's good to make the expectations explicit here as well.

Reply:
Makes sense, applied!